ext/sockets/conversions.c

```diff
@@ -1016,7 +1016,7 @@ static void to_zval_read_control_array(const char *msghdr_c, zval *zv, res_conte
 	uint32_t		i = 1;
 
 	/*if (msg->msg_flags & MSG_CTRUNC) {
-		php_error_docref0(NULL, E_WARNING, "The MSG_CTRUNC flag is present; will not "
+		php_error_docref(NULL, E_WARNING, "The MSG_CTRUNC flag is present; will not "
 				"attempt to read control messages");
 		ZVAL_FALSE(zv);
 		return;
```

Comment:
Should I remove this dead code?

Reply:
In general, you want to keep the PR focused on what you are doing, if you happen to see dead code/white space or other gremlins, open a quick PR separately for that, and someone will hopefully merge it quickly.